src/components/network/overloaded-lines-view.js

```diff
@@ -51,6 +51,7 @@ const useStyles = makeStyles((theme) => ({
 
 const OverloadedLinesView = (props) => {
     const [lines, setLines] = useState(null);
+    const [lineLoaded, setLineLoaded] = useState(false);
```

Comment:
linesLoaded for state name and setLinesLoaded for setter name

Reply:
done&done